client/landing/stepper/declarative-flow/internals/components/survery-manager/index.tsx

```diff
@@ -1,29 +1,20 @@
 import { SITE_MIGRATION_FLOW, HOSTED_SITE_MIGRATION_FLOW } from '@automattic/onboarding';
-import { type ComponentType, Suspense } from 'react';
+import { Suspense } from 'react';
 import AsyncMigrationSurvey from '../../steps-repository/components/migration-survey/async';
 import { Flow } from '../../types';
 import { DeferredRender } from '../deferred-render';
 
-const FLOW_SURVEYS = new Map< string, ComponentType >( [
-	[ SITE_MIGRATION_FLOW, AsyncMigrationSurvey ],
-	[ HOSTED_SITE_MIGRATION_FLOW, AsyncMigrationSurvey ],
-] );
+const MIGRATION_FLOWS = new Set< string >( [ SITE_MIGRATION_FLOW, HOSTED_SITE_MIGRATION_FLOW ] );
 
 const SurveyManager = ( { disabled = false, flow }: { disabled?: boolean; flow?: Flow } ) => {
-	if ( disabled || ! flow ) {
-		return null;
-	}
-
-	const SurveyComponent = FLOW_SURVEYS.get( flow.name );
-
-	if ( ! SurveyComponent ) {
+	if ( disabled || ! flow || ! MIGRATION_FLOWS.has( flow.name ) ) {
```

Comment:
import { isNewSiteMigrationFlow } from '@automattic/onboarding';

isNewSiteMigrationFlow(flow.name)

The hosted-site migration is just a variant, so the name will always be SITE_MIGRATION.

BTW, I am working now to unify both migration flows,, and I probably will rename the `isNewSiteMigrationFlow` to > isSiteMigration

Reply:
Thanks for the pointer to that helper method. I updated in b97a5f9